pkg/daemon/ceph/agent/flexvolume/manager/ceph/manager.go

```diff
@@ -161,6 +161,18 @@ func (vm *VolumeManager) Attach(image, pool, id, key, clusterNamespace string) (
 	}
 }
 
+func (vm *VolumeManager) Resize(image, pool, clusterNamespace string, size uint64) error {
```

Comment:
I'm struggling with terminology here. Resize suggests grow and shrink, but expand suggests only grow. If this only allows growing, what happens if this gets negative valaues?